modules/geotiff/src/loaders.ts

```diff
@@ -0,0 +1,68 @@
+import { fromArrayBuffer } from "geotiff";
+import type {LoaderWithParser, LoaderOptions} from '@loaders.gl/loader-utils';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+// Type definitions
+type GeoTiffData =  {
+    bounds: number[],
+    width: number,
+    height: number,
+    imageData: ImageData,
+}
+
+interface GeoTiffLoaderOptions extends LoaderOptions {
+    enableAlpha: boolean;
+}
```

Comment:
Nit: Follow pattern set by other loaders
```suggestion
export type GeoTiffLoaderOptions = LoaderOptions & {
  geotiff?: {
    enableAlpha: boolean;
  }
}
```